ratis-proto/src/main/proto/Raft.proto

```diff
@@ -413,6 +413,11 @@ message TransferLeadershipRequestProto {
   RaftPeerProto newLeader = 2;
 }
 
+// snapshot request
+message SnapshotRequestProto {
```

Comment:
Let's call it SnapshotManagementRequestProto.  We should also rename SnapshotRequest to SnapshotManagementRequest.